sky/packages/sky/lib/src/widgets/input.dart

```diff
@@ -22,40 +22,42 @@ class Input extends StatefulComponent {
 
   Input({
     GlobalKey key,
+    this.value : '',
     this.placeholder,
     this.onChanged,
     this.keyboardType : KeyboardType_TEXT
   }): super(key: key);
 
   int keyboardType;
+  String value;
   String placeholder;
   StringValueChanged onChanged;
 
-  String _value = '';
```

Comment:
I changed to _value as you suggested, but added a getter: what would be the problem with allowing reading the value?